docs/UserGuide.md

```diff
@@ -137,6 +137,35 @@ Examples:
 * `delete /event 1 ` deletes the 1st event in the event list.
 * `delete /event 10 ` deletes the 10th event in the event list.
 
+### Add Member to an Event: `add /member /event`
+
+Adds a member to an event. 
+
+Format: `add /member MEMBER_INDEX /event EVENT_INDEX`
+
+* Add the member at the specified `MEMBER_INDEX` to the event at the specified `EVENT_INDEX`.
+* The member/event index refers to the index number shown in the displayed member/event list.
+* The member/event index **must be a positive integer** that is within the range of the length of the member/event list.
```

Comment:
Should we follow the `MEMBER_INDEX` and `EVENT_INDEX` format instead of "member/event index" to ensure consistency